src/actions/toastAction.js

```diff
@@ -0,0 +1,33 @@
+import { NEW_TOAST, CLEAR_TOAST } from './actionTypes';
```

Comment:
@devdbrandy Are you still using toasters?

Reply:
Currently I'm not. But since we have toasters configured with the system, its a cleaner way to send a toast message right from an action. So instead of using a toaster directly from our action, we simply dispatch a `toastAction`.

Reply:
@devdbrandy I don't think there is a need for a toaster action though. since you are not displaying a message for the user anymore.
<img width="757" alt="Screenshot 2019-05-01 at 8 54 53 PM" src="https://user-images.githubusercontent.com/38490848/57039022-6b4ffb80-6c53-11e9-8030-5a0bdf95d6ab.png">